### DISCUSSION TITLE
Quarto Kickstarter Insert mode

### DISCUSSION THREAD

**Discussion by ivanuricardo** [quarto-dev/quarto-nvim#96] [Q&A]
Hello,

I have been trying the kickstarter and it works very nicely! However, I have some existing qmd files and when I try editing those, quarto nvim acts strange.

 For instance, I edit the files in insert mode, but at the bottom, I keep getting "Press ENTER or type command to continue". Additionally, typing a space bar does not insert a space bar, but rather I have to press spacebar twice in order to physically have a space bar.

When I try on a new qmd file, I can enter insert mode no problem, but the bar at the bottom does not say what type of file I am working on or what line I am on. Any tips on how to fix this? Thanks!

**Comment by jmbuhr** [comment]
Can you provide an example qmd file in which this happens for you? I am not able to reproduce any of the two behaviors.

**Reply by ivanuricardo** [reply]
Ok I used the LazyVim configuration file and it all works fine. Must have been something on my end in that case :^)
